src/battle_util.c

```diff
@@ -2478,15 +2457,16 @@ u8 AbilityBattleEffects(u8 caseID, u8 bank, u8 ability, u8 special, u16 moveArg)
             {
                 if (gBattleMons[i].ability == ABILITY_TRACE && (gStatuses3[i] & STATUS3_TRACE))
                 {
-                    u8 opposite = (GetBankIdentity(i) ^ 1) & 1;
-                    u8 target1 = GetBankByIdentity(opposite);
-                    u8 target2 = GetBankByIdentity(opposite + 2);
+                    u8 target2;
```

Comment:
Does this still match if these variables are declared at the top of the function rather than mid-function?

Reply:
Yeah